### PR TITLE
Adding User Settings Page

### PR DESCRIPTION
**Pull request by BarbzCodez**
## Changes
- Added the skeleton to the User Settings Page
- Added a link to the user settings to in the `user profile picture > Settings`
- Changed created index for the themes

## Preview
image

### CODE REVIEW

File: client/src/pages/UserSettings/styles.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import { List } from '@mui/material';
+
+export const SettingsList = styled(List)({
+  width: '45vw',
+  minWidth: '350px',
+  maxWidth: '600px',
+});
+
+export const SettingsStack = styled(Stack)({
+  background: 'rgb(53, 52, 73, 0.8)',
```
Comment (VictoriaKGN) [inline client/src/pages/UserSettings/styles.ts:11]: you can access this color from `theme.palette.secondary.main`

**Review by VictoriaKGN — COMMENTED** [review]
looks good, although it looks funky on my 15-inch laptop. might need to play with the paddings/margins/spacings so that we can accommodate for smaller screens.

**Review by VictoriaKGN — APPROVED** [review]
image

File: client/src/pages/UserSettings/components/styles.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  width: '50vw',
+  minWidth: '350px',
+  maxWidth: '600px',
+  justifyContent: 'space-between',
+  spacing: '1rem',
+  alignItems: 'center',
+  flexDirection: 'row',
+  paddingLeft: '0.5rem',
+  paddingRight: '0.5rem',
+  // padding: '0.5rem',
```
Comment (dlarocque) [inline client/src/pages/UserSettings/components/styles.ts:47]: can u remove commented out code

**Review by dlarocque — APPROVED** [review]

**Pull request merged**